code/modules/food_and_drinks/recipes/soup_mixtures.dm

```diff
@@ -551,7 +551,10 @@
 	drink_type = VEGETABLES | FRUIT // ??
 
 /datum/chemical_reaction/food/soup/tomatosoup
-	required_reagents = list(/datum/reagent/water = 50)
+	required_reagents = list(
+		/datum/reagent/water = 50,
```

Comment:
```suggestion
		/datum/reagent/water = 45,
```

Reply:
Given that you always put 50u of water in a cookpot when you fill it this is just going to end up with leftover water in the soup mix, but I'll do it.